src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/backpacks/ContainmentBackpack.java

```diff
@@ -0,0 +1,33 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.backpacks;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactive;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This implementation of {@link SlimefunBackpack} for Radioactive Items.
+ *
+ * @author TheSilentPro
+ *
+ */
+public class ContainmentBackpack extends SlimefunBackpack {
+
+    public ContainmentBackpack(int size, Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
+        super(size, category, item, recipeType, recipe);
+    }
+
+    @Override
+    public boolean isItemAllowed(@Nonnull ItemStack item, @Nonnull SlimefunItem itemAsSlimefunItem) {
```

Comment:
Don't just add random annotations here, SlimefunItem should be nullable.
Also these annotations don't exist in the super method, so you should add them there too.

Reply:
Nonnull is to make sure that object is not null correct?
Also added because of: https://github.com/Slimefun/Slimefun4/pull/2343#discussion_r493004998

Reply:
No, those are just **Annotations**. they annotate what behaviour is intended.
Developers use these to communicate what to do with something, `@Nonnull` communicates "Hey, I made sure that this will never be null", `@Nullable` means "This can definitely be null sometimes, so make sure to catch that!".

Blindly adding them without precautions sends of the wrong signals, particularly because you are simply overridding the method. These should be added to the actual method behind it too. And like I said, this method is invoked with SlimefunItem being null sometimes.